locales/de/translation.json

```diff
@@ -32,50 +32,80 @@
-    "unlock-key": "Freischaltschlüssel",
-    "enter-unlock-key": "{{moduleId}} ist lizenzgeschützt. Bitte geben Sie den Freischaltschlüssel ein!",
-    "unlock-failed": "Freischalten fehlgeschlagen! Bitte geben Sie den korrekten Freischaltschlüssel ein."
+    "removal": "Entfernen",
+    "update-data": { 
+      "repo-data-last-updated": "Internetarchivdaten wurden zuletzt am {{date}} aktualisiert.",
+      "update-now": "Jetzt aktualisieren",
+      "updating-repository-data": "Aktualisiere Internetarchivdaten! Das dauert ein paar Sekunden ...",
+      "update-failed": "Update failed",
+      "update-repository-data-failed": "Aktualisieren der Internetarchivdaten fehlgeschlagen. Sind Sie offline?"
+    },
+    "step-languages": {  
+      "select-language": "Please choose at least one of the following languages.",
+      "total-modules-language": "Next to each language is the number of {{module_type}} available in the SWORD repositories.",
+      "loading-languages": "Lade Sprachen ...",
+      "app-system-languages": "Existing Languages and System Languages",
+      "bible-languages": "Bible Languages",
+      "most-spoken-languages": "Most Spoken Languages",
+      "historical-languages": "Historical Languages",
+      "other-languages": "Other Languages",
+      "language-search-results": "Matches Found"
+    },
+    "step-repositories": {
+      "select-repository": "Select at least one of the following SWORD repositories.",
+      "total-modules-repo": "Next to each repository is the number of {{module_type}} available from the repository.",
+      "what-is-repository": "A repository is an internet storage location that contains a set of {{module_type}}",
+      "loading-repositories": "Lade Internetarchive ...",
+      "repo-recommended": "This repository is a good place to start",
+      "more-repo-information-needed": "Sie benötigen weitere Informationen zu diesen Internetarchiven?<br/>Werfen Sie einen Blick auf das <a class='external' href='https://wiki.crosswire.org/Official_and_Affiliated_Module_Repositories'>CrossWire Wiki</a> (Englisch)."
+    },
+    "step-modules": {  
+      "select-module": "Please choose {{module_type}} to install.",
+      "module-display-preferences": "Filter preferences:",
+      "module-with": "{{module_type, capitalize}} with {{feature_type, instrumental}}",
+      "module-already-installed": "This {{module_type, singular}} is already installed",
+      "show-module-info":"Show info",
+      "show-detailed-module-info": "Press on the information icon for more details about the {{module_type, singular}}."
+    },
+    "step-install": {
+      "installing-modules": "Installing selected {{module_type, accusative}}",
+      "installing": "Installiere",
+      "it-takes-time-to-install": "Note that it takes some time for each {{module_type, singular-genitive}} to be downloaded and then installed.",
```

Comment:
In my mind, this is accusative ... (the module type is the object in this sentence).

Reply:
@zhuiks Any feedback on this one?